### PR TITLE
Clamp tools

### PR DESCRIPTION
**Pull request by PLally**
wrap turret and explosive setup functions in order to clamp their values

### CODE REVIEW

File: lua/tool_balance/wire/explosive.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    damage  = {0, 20},
+    radius  = {0, 500},
+    delayreloadtime = {5,math.huge}
+}
+
+local WIRE_EXPLOSIVE = scripted_ents.GetStored( "gmod_wire_explosive" ) 
+
+local function wrapWireExplosive() 
+    local setup = WIRE_EXPLOSIVE.t.Setup
+    WIRE_EXPLOSIVE.t.Setup = function(self, key, damage, delaytime, removeafter, radius, affectother, otaffected, delayreloadtime, maxhealth, bulletproof, explosionproof, fallproof, explodeatzero, resetatexplode, fireeffect, coloreffect, invisibleatzero )
```
Comment (PLally) [inline lua/tool_balance/wire/explosive.lua]: this line is too long...

File: lua/tool_balance/tools/base/turret.lua [hunk 2]
```diff
@@ -0,0 +1,39 @@
+-- base/turret
+
+-- config
+local config = { 
+    delay   = {0.5, math.huge},
```
Comment (brandonsturgeon) [inline lua/tool_balance/tools/base/turret.lua]: I think you may as well just go full on with this:

```lua
delay = { min = 0.5, max = math.huge }
```
etc.

File: lua/tool_balance/tools/base/turret.lua [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+local function wrapTurret() 
+    local TURRET =  scripted_ents.GetStored( "gmod_turret" ).t
+    
+    TURRET.SetDelay = clampFunction(TURRET.SetDelay, {values.delay})
+    TURRET.SetDamage = clampFunction(TURRET.SetDamage, {values.damage})
+    TURRET.SetForce = clampFunction(TURRET.SetForce, {values.force})
+    TURRET.SetNumBullets = clampFunction(TURRET.SetNumBullets, {values.numbullets})
+    TURRET.SetSpread = clampFunction(TURRET.SetSpread , {values.spread})
+
+    print("[cfc_tool_balance] base/turret loaded")
```
Comment (brandonsturgeon) [inline lua/tool_balance/tools/base/turret.lua]: Can we capitalize this?:

```lua
print("[CFC_Tool_Balance] base/turret loaded")
```

**Comment by PLally** [comment]
Tested this to confirm it still works.

**Pull request merged**